core/domain/wipeout_service.py

```diff
@@ -396,6 +396,21 @@ def remove_user_from_activities_with_associated_rights_models(
         rights_manager.deassign_role_for_exploration(
             user_services.get_system_user(), exp_id, user_id)
 
+    if not use_user_subscriptions_ids:
```

Comment:
Is there a reason why we don't always just have use_user_subscription_ids=False (if it is more robust that way)?

It seems to me that using the subscriptions thing is fragile. E.g. we may give creators the option to unsubscribe from specific entities in the future, just as GitHub allows muting threads...

Reply:
Done. It was needed in order to speed up the pre_deletion part but since we defer that now it is fine to retrieve the models.